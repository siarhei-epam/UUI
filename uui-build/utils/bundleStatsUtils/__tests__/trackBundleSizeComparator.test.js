const { compareBundleSizes } = require('../trackBundleSizeComparator.js');

const TestData = {
    baseLineSizes:{ testApp1: 100, testApp2: 200 },
    newSizes:{ testApp1: 120, testApp2: 210 },
}

describe('trackBundleSizeComparator', () => {
    it('should create comparison result report', () => {
        const result = compareBundleSizes({ baseLineSizes: TestData.baseLineSizes, newSizes: TestData.newSizes });
        expect(result).toMatchSnapshot();
    });
})
