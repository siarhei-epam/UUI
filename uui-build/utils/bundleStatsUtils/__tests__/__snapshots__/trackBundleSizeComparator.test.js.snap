// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`trackBundleSizeComparator should create comparison result report 1`] = `
Object {
  "testApp1": Object {
    "baseLineSize": 0.1,
    "diffLabel": "+0.02",
    "size": 0.12,
    "thresholdLabel": "0.09 - 0.11",
    "withinThreshold": false,
  },
  "testApp2": Object {
    "baseLineSize": 0.2,
    "diffLabel": "+0.01",
    "size": 0.21,
    "thresholdLabel": "0.18 - 0.22",
    "withinThreshold": true,
  },
}
`;
